caching-service/src/main/java/org/zowe/apiml/caching/service/vsam/VsamStorage.java

```diff
@@ -38,28 +39,41 @@ public VsamStorage(VsamConfig vsamConfig, VsamInitializer vsamInitializer) {
         this.vsamConfig = vsamConfig;
         if (evictionStrategy.equals(Strategies.REJECT.getKey())) {
             strategy = new RejectStrategy();
-        } else if (evictionStrategy.equals(Strategies.REMOVE_OLDEST.getKey())) {
-//           strategy = new RemoveOldestStrategy(storage);
```

Comment:
Remove this altogether

Reply:
This is not present anymore, were you maybe looking at old commit?